project/cmake/modules/FindPython.cmake

```diff
@@ -5,7 +5,7 @@
 # PYTHON_INCLUDE_DIRS - the python include directory
 # PYTHON_LIBRARIES - The python libraries
 
-if(PKG_CONFIG_FOUND AND NOT CMAKE_CROSSCOMPILING)
+if(PKG_CONFIG_FOUND AND (NOT CMAKE_CROSSCOMPILING OR APPLE))
```

Comment:
what I was trying to say above is that _all platforms_ but windows should work with pkgconfig.